Index: black/setup.py
===================================================================
--- black.orig/setup.py
+++ black/setup.py
@@ -1,5 +1,6 @@
 # Copyright (C) 2020 Łukasz Langa
 from setuptools import setup
+from _black_version import version as __version__
 import sys
 import os
 
@@ -47,10 +48,7 @@ else:
 
 setup(
     name="black",
-    use_scm_version={
-        "write_to": "src/_black_version.py",
-        "write_to_template": 'version = "{version}"\n',
-    },
+    version=__version__,
     description="The uncompromising code formatter.",
     long_description=get_long_description(),
     long_description_content_type="text/markdown",
Index: black/docs/conf.py
===================================================================
--- black.orig/docs/conf.py
+++ black/docs/conf.py
@@ -13,6 +13,7 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
 from pathlib import Path
+from _black_version import version as __version__
 import re
 import string
 from typing import Callable, Dict, List, Optional, Pattern, Tuple, Set
@@ -207,7 +208,8 @@ author = "Łukasz Langa and contributors
 
 # Autopopulate version
 # The version, including alpha/beta/rc tags, but not commit hash and datestamps
-release = get_distribution("black").version.split("+")[0]
+# release = get_distribution("black").version.split("+")[0]
+release = __version__
 # The short X.Y version.
 version = release
 for sp in "abcfr":
